enhancement: Created Create Post Page

ckhorra1 opened the pull request:
Ari and Cyrus created the Create Post Page. In this page there are two text fields and a submit button. Once the user enters in the image url and post tile in both textfields they will click the submit button. Once clicked the button will return a message indicating if the creation was successful or not. Issue #20

ckhorra1 committed:
feat: Created Create Post Page

Ari and Cyrus created the Create Post Page. In this page there are two text fields and a submit button. Once the user enters in the image url and post tile in both textfields they will click the submit button. Once clicked the button will

zxkevin reviewed (changes requested):
Should this PR not include changes to App.js to route users to this page upon loading in?

ckhorra1 committed:
enhancement: Updated app.js

Updated app.js and removed clutter and imports that were not needed. Implemented the react-router-dom to route the page to the Create Post page.

zxkevin commented on packages/hackmcx-client/src/App.js:
```diff
@@ -1,26 +1,20 @@
+		<div className='App'>
+			<Router>
+				<Routes>
+					<Route exact path='/' exact element={<CreatePost />} />
```
Please change to: `<Route exact path='/CreatePost' exact element={<CreatePost />} />`

zxkevin reviewed (changes requested):
Check comments

ckhorra1 committed:
Updated app.js

zxkevin commented on packages/hackmcx-client/package.json:
```diff
@@ -3,14 +3,19 @@
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.25.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
```
Need to include react-router-dom

zxkevin reviewed (changes requested):
Update package/package-lock, check comments

ckhorra1 committed:
Updated pack/lock json files

Installed the react-router-dom which updated both the package-lock and package json file.

zxkevin reviewed (approved):
Looks good!

ckhorra1 committed:
Merge branch 'main' into enhancement-cyrus-ari-createpostpage
